src/EFCore.PG/Internal/NpgsqlOptions.cs

```diff
@@ -47,14 +47,9 @@ public class NpgsqlOptions : INpgsqlOptions
         [NotNull]
         public virtual IReadOnlyList<RangeMappingInfo> RangeMappings { get; private set; }
 
-        /// <inheritdoc />
-        [NotNull]
-        public virtual IReadOnlyList<NpgsqlEntityFrameworkPlugin> Plugins { get; private set; }
-
         public NpgsqlOptions()
         {
             RangeMappings = new RangeMappingInfo[0];
-            Plugins = new NpgsqlEntityFrameworkPlugin[0];
```

Comment:
As above, you can expression-body this constructor.

Reply:
Done